cfme/utils/appliance/__init__.py

```diff
@@ -366,6 +366,11 @@ def _version_tostr(x):
             for k in set(self.CONFIG_MAPPING.values())
             if k in self.__dict__})
 
+    @property
+    def is_configured(self):
+        """ Check if the appliance is configured or unconfigured."""
+        return self.db.has_database
```

Comment:
If the DB is configured shouldn't we also check if the DB `has_tables` 

```suggestion
        return self.db.has_database and self.db.has_tables
```